MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -63,14 +63,14 @@ public BaseNfoParser(ILogger logger, IConfigurationManager config, IProviderMana
         /// <exception cref="ArgumentException"><c>metadataFile</c> is <c>null</c> or empty.</exception>
         public void Fetch(MetadataResult<T> item, string metadataFile, CancellationToken cancellationToken)
         {
-            if (item == null)
+            if (item.Item == null)
```

Comment:
```suggestion
            if (item?.Item == null)
```

Reply:
`item` shouldn't be `null` as nullable reference types is turned on for MediaBrowser.XbmcMetadata